Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -569,6 +569,220 @@ class UpstreamIntegrationTests: TestBase {
         assertExpectedEvents(ignoreUnexpectedEvents: true)
     }
 
+    // Tests datastream config overrides
+    func testSendEvent_withValidConfigOverrides_receivesExpectedNetworkResponse() {
+        // Setup
+        // Note: test constructs should always be valid
+        let interactNetworkRequest = NetworkRequest(urlString: createURLWith(locationHint: edgeLocationHint), httpMethod: .post)!
+        // Setting expectation allows for both:
+        // 1. Validation that the network request was sent out
+        // 2. Waiting on a response for the specific network request (with timeout)
+        networkService.setExpectationForNetworkRequest(networkRequest: interactNetworkRequest, expectedCount: 1)
+        let configOverrides = ["com_adobe_experience_platform": [
+                                    "datasets": [
+                                        "event": [
+                                            "datasetId": "do_not_delete_dataset2_e2e_testing_edge_konductor_configoverride"
```

Comment:
this should be the datasetId and not the table name right?

Reply:
Yes and the tests still passed 

![](https://media.giphy.com/media/TJawtKM6OCKkvwCIqX/giphy.gif)

Reply:
Is this a bug, are you tracking this down?